client/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-unresolved */
+import React from 'React';
```

Comment:
shouldn't 'React' be 'react' lowercase?

Reply:
i don't think it's case sensitive there, but sure I'll change it.